api/api.py

```diff
@@ -1,7 +1,105 @@
-from flask import Blueprint
+import requests
+import os
+import json
+import sqlite3 as sql
+import config
 
-api_bp = Blueprint('api_bp', __name__)
+from flask import Blueprint, request, jsonify
+from dotenv import load_dotenv
+from stocks.stocks import get_stock_quote, calculate_percent_change, validate_stock_symbol, insert_stock_tracker, filter_tracked_stocks_details
 
-@api_bp.route('/')
+load_dotenv()
+
+api_bp = Blueprint("api_bp", __name__)
```

Comment:
WIP: We can work on a better name in the future I feel

This blueprint right now is kind of a "catch-all"